server-phoenix/test/helios_web/schema/resolvers/event_collection_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule HeliosWeb.Schema.Resolvers.EventCollectionTest do
+  use HeliosWeb.ConnCase, async: true
+  use ExUnit.Case
+  alias Helios.Event
+  alias Helios.Repo
+
+  test "getting authors by # of contributions", %{conn: conn} do
+    conn =
+      conn
+      |> put_req_header("content-type", "application/json")
+      |> put_req_header("x-github-event", "pull_request")
+
+    conn
+    |> post(
+      Routes.github_path(conn, :handle),
+      %{
+        action: "opened",
+        pull_request: %{
+          url: "https://api.github.com/repos/Codertocat/Hello-World/pulls/1",
+          id: "279147437",
+          user: %{login: "RVRX"}
+        }
+      }
+    )
```

Comment:
You could create a method that just calls `Repo.insert` for an `Event` created by passed in args. No need to go to the controller for this.

Reply:
good suggestion! fixed and looking a lot cleaner now, added an additional test for some more coverage as well